river/stats/cov.py

```diff
@@ -68,12 +68,16 @@ def __init__(self, ddof=1):
         self.mean_y = stats.Mean()
         self.cov = 0
 
+    @property
```

Comment:
I got so used to `var.mean.n` that I didn't realize that `var.n` is much clearer! Good stuff